### ISSUE TITLE
`kemonopartydownloader.py`: add youtube links support

### ISSUE THREAD

**Issue by mrpapersonic** [mrpapersonic/codedump#2]
ya

**Comment by mrpapersonic** [comment]
just a note: we should probably also dump the json, because this should be treated as an archiving tool (possible rename?????)

**Comment by mrpapersonic** [comment]
sidenote: it might be worth just completely dumping this project and moving over to gallery-dl for this kind of stuff, but we have some features that gallery-dl lacks (automatically parsing gdrive, dropbox, and (to be) youtube links, and some other stuff)

**Comment by mrpapersonic** [comment]
okay, I'll officially deprecate this: gallery-dl is honestly way better for this sort of stuff. it doesn't AUTOMATICALLY parse gdrive links, but it has options to dump JSON, which can then be parsed for links and stuff to gdrive and yt and the like

tl;dr stop using this script lmao
